process_report/tests/e2e/test_e2e_pipeline.py

```diff
@@ -124,9 +126,11 @@ def _prepare_pipeline_execution(
         str(test_files["test_prepay_contacts.csv"]),
     ]
 
-    # Set up environment
+    # Environment setup for subprocess execution
     env = os.environ.copy()
-    env["CHROME_BIN_PATH"] = CHROME_BIN_PATH
+    # Fallback ensures test works even when CI environment doesn't set Chrome path
+    if "CHROME_BIN_PATH" not in env:
+        env["CHROME_BIN_PATH"] = "/usr/bin/chromium"
```

Comment:
You could instead do this:
```suggestion
env.setdefault("CHROME_BIN_PATH", "/usr/bin/chromium")
```